ui/src/components/admin/sermonNotes/AdminSermonNotesContainer.js

```diff
@@ -0,0 +1,378 @@
+        isClosable: true,
+      });
+      setIsLoading(false);
+    }
+  };
+
+  const onPublish = async (e) => {
+    try {
+      setIsLoading(true);
+      const sermonNotesId = e.target.value;
+      const sermonNotesData = sermonNotesList.find(
+        (sermonNote) => sermonNote.sermonId === sermonNotesId
+      );
+      const { status } = await axios.put('/api/sermon-notes-parent/update', {
+        sermonId: sermonNotesId,
+        title: sermonNotesData.title,
+        speaker: sermonNotesData.speaker,
+        date: sermonNotesData.date,
+        originalContent: sermonNotesData.originalContent,
+        sermonLink: sermonNotesData.sermonLink,
+        serviceType: sermonNotesData.serviceType,
+        passage: sermonNotesData.passage,
+        isPublished: !sermonNotesData.isPublished,
+      });
+
+      if (status === 200) {
+        toast({
+          description: 'Sermon Notes has been updated',
+          status: 'success',
+          duration: 8000,
+          isClosable: true,
+        });
+      } else {
+        toast({
+          description: 'There was an issue with the request',
+          status: 'error',
+          duration: 8000,
+          isClosable: true,
+        });
+      }
+
+      await fetchSermonNotes();
+      setIsLoading(false);
+    } catch (err) {
+      console.log(err);
+      toast({
+        description:
+          'There was an issue with the request, please talk to t3ch support',
+        status: 'warning',
+        duration: 8000,
+        isClosable: true,
+      });
+      setIsLoading(false);
+    }
+  };
+
+  const onEdit = async (e) => {
+    setIsEditorOpen(true);
+    try {
+      const { data, status } = await axios.get('/api/sermon-notes-parent/get', {
+        params: {
+          sermonId: e.target.value,
+        },
+      });
+
+      if (status !== 200) {
+        toast({
+          description:
+            'There was an issue with the request, please talk to a t3ch support',
+          status: 'warning',
+          duration: 8000,
+          isClosable: true,
+        });
+      }
+      setIsEditorOpen(true);
+      setActionOnEditor('edit');
+      setEditSermonNotesData(data[0]);
+      setIsLoading(false);
+    } catch (err) {
+      console.log(err);
+      toast({
+        description:
+          'There was an issue with the request, please talk to t3ch support',
+        status: 'warning',
+        duration: 8000,
+        isClosable: true,
+      });
+      setIsLoading(false);
+    }
+  };
+
+  useEffect(() => {
+    fetchSermonNotes();
+  }, [isEditorOpen]);
+
+  return (
+    <Container maxW="container.xl">
+      <Heading as="h1" size="xl" pb={3}>
+        Sermon Notes Manager
+      </Heading>
+      <Stack direction="row" mb={5}>
+        <Button
+          colorScheme="blue"
+          variant={isEditorOpen ? 'solid' : 'outline'}
+          borderColor="#3182CE"
+          borderWidth={3}
+          size="lg"
+          disabled={isPublishDisabled}
```

Comment:
this is doing reference to the function. But you want to call the function so needs bracket